fdbserver/DataDistribution.actor.cpp

```diff
@@ -765,6 +768,12 @@ struct DDTeamCollection : ReferenceCounted<DDTeamCollection> {
 				} else {
 					self->medianAvailableSpace = SERVER_KNOBS->MIN_AVAILABLE_SPACE_RATIO;
 				}
+				if (self->medianAvailableSpace < SERVER_KNOBS->TARGET_AVAILABLE_SPACE_RATIO) {
+					TraceEvent(SevWarn, "DDTeamMedianAvailableSpaceTooSmall")
+					    .detail("MedianAvailableSpaceRatio", self->medianAvailableSpace);
```

Comment:
can we include TARGET_AVAILABLE_SPACE_RATIO here as well?
Operator won't need to go to different places to check if the knob is changed.

Reply:
Done.